.github/workflows/trivy.yml

```diff
@@ -0,0 +1,43 @@
+# Runs linter for Docker files
+name: Trivy
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+    paths:
+      - '.github/docker/*Dockerfile'
+      - '.github/workflows/trivy.yml'
```

Comment:
Do we scan dockerfiles or dockerfiles and images?

Reply:
in this workflow we only scan for misconfigurations (with command `trivy config`). Docs: https://aquasecurity.github.io/trivy/v0.49/docs/scanner/misconfiguration/

If you want to scan images that's another story 😉 (for now, we don't build them)